run.sh

```diff
@@ -0,0 +1,19 @@
+
+set -e
+
+source /opt/adapteva/esdk/setup.sh
+
+if [ ! -f /app/installed.txt ]; then
+	echo "Building FFT correlation"
```

Comment:
Yes. The CL linker from COPRTHR looks for /dev/epiphany at build time, so it fails on the build server.